src/main.rs

```diff
@@ -15,8 +16,9 @@ use serde_derive::Serialize;
 
 /// Generate a random string.
 fn generate_random_string(n: usize, sigma: usize) -> Vec<u8> {
+    let mut rng = rand::rngs::StdRng::seed_from_u64(213456);
```

Comment:
```suggestion
    let mut rng = rand::rngs::StdRng::seed_from_u64(31415);
```
I always use `pi` as a seed so it's clear that the number is random ;)

Reply:
https://docs.rs/rand/latest/rand/rngs/struct.StdRng.html

> The algorithm is deterministic but should not be considered reproducible due to dependence on configuration and possible replacement in future library versions. For a secure reproducible generator, we recommend use of the [rand_chacha](https://crates.io/crates/rand_chacha) crate directly.


do you think that's sufficient? Otherwise we should use `rand_chacha` directly.

Reply:
ooo nice catch. Yea I'll sub it out for `rand_chacha`, thanks